packages/react/src/utils/positioner/Popper.tsx

```diff
@@ -58,18 +78,59 @@ const Popper: React.FunctionComponent<PopperProps> = props => {
 
   const popperRef = React.useRef<PopperJS>()
   const contentRef = React.useRef<HTMLElement>(null)
+
   const latestPlacement = React.useRef<PopperJS.Placement>(proposedPlacement)
   const [computedPlacement, setComputedPlacement] = React.useState<PopperJS.Placement>(
     proposedPlacement,
   )
 
-  const computedModifiers: PopperJS.Modifiers = React.useMemo(
+  const hasScrollableElement = React.useMemo(() => {
+    const scrollParentElement = getScrollParent(contentRef.current)
+
+    return scrollParentElement !== scrollParentElement.ownerDocument.body
+  }, [contentRef])
```

Comment:
I don't think that using `Ref` as `useMemo` dependency works.

Reply:
As discussed in offline added a comment there.